src/firefly/js/util/WebUtil.js

```diff
@@ -286,4 +287,42 @@ export function deepDiff(o1, o2, p) {
         }
         console.groupEnd();
     }
-}
\ No newline at end of file
+}
+
+
+/*----------------------------< COOKIES ----------------------------*/
+export function setCookie(name, value, options = {}) {
+    var str = `${encodeURIComponent(name)}=${encodeURIComponent(value)}`;
+
+    if (value == null) options.maxage = -1;
```

Comment:
not that trivial. Since the code was an example on the web it made some discussion between Loi and I about what the right thing to do.  We settled with lodash isNil